doc/articles/features/windows-media-capture.md

```diff
@@ -0,0 +1,62 @@
+---
+uid: Uno.Features.Capture
```

Comment:
The new doc needs to be added to Table of contents

Reply:
@MartinZikmund Thanks! Would it be possible to add a CI validation, to warn/fail the contributor, if the page is not added in toc?